internal/pkg/v2/application/dto/v2dot0/error/response.go

```diff
@@ -0,0 +1,29 @@
+/*******************************************************************************
+ * Copyright 2020 Dell Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *******************************************************************************/
+
+package error
+
+import dtoBase "github.com/edgexfoundry/edgex-go/internal/pkg/v2/application/dto/v2dot0/base"
+
+// Response defines the content of the generic error DTO.
+type Response struct {
+	dtoBase.Response `json:",inline"`
+}
+
+// NewResponse is a factory function that returns an initialized Response struct.
+func NewResponse(baseResponse *dtoBase.Response) *Response {
```

Comment:
How is this adding anything error wise to the base response?

Reply:
Strictly speaking, it isn't.  However, in the OpenAPI specification the ErrorResponse is a separate entity from the BaseResponse.  This is what's being modeled in the code.  It provides for possible future changes to ErrorResponse without incurring the cost of refactoring the code base to add it in a later revision.